protoc-builder/Dockerfile.jsonschema

```diff
@@ -1,17 +1,15 @@
-# TODO: use golang image
-FROM alpine@sha256:56fa17d2a7e7f168a043a2712e63aed1f8543aeafdcee47c58dcffe38ed51099
-RUN apk add --update protoc protobuf-dev go git
+FROM golang:1.23.4-alpine@sha256:c23339199a08b0e12032856908589a6d41a0dab141b8b3b21f156fc571a3f1d3 AS jsonschema-builder
 
 ADD hack/jsonschema/go.* hack/jsonschema/tools.go tools/
 
 # the specific versions of these tools are in hack/go.mod so that Dependabot can bump them for updates
 RUN cd tools && go build --trimpath -o /usr/local/bin/protoc-gen-jsonschema github.com/chrusty/protoc-gen-jsonschema/cmd/protoc-gen-jsonschema
 
```

Comment:
To confirm, this is no longer needed?

Reply:
this now pulls protoc like the other languages rather than installing a version from alpine.